tests/imfile-ignore-old-file-1.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+# Don't read a file with old timestamp
+echo [imfile-ignore-old-file-1.sh]
+. ${srcdir:=.}/diag.sh init
+. $srcdir/diag.sh check-inotify
+generate_conf
+add_conf '
+module(load="../plugins/imfile/.libs/imfile")
+input(type="imfile"
+      File="./'$RSYSLOG_DYNNAME'.input"
+      ignoreolderthan="604800"
+      Tag="file:"
+      ruleset="ruleset")
+template(name="outfmt" type="string" string="%msg:F,58:2%\n")
+ruleset(name="ruleset") {
+	action(type="omfile" file="'$RSYSLOG_OUT_LOG'" template="outfmt")
+}
+'
+
+# create a log for testing
+./inputfilegen -m 1000 > ${RSYSLOG_DYNNAME}.input
+
+touch -m -t 201806010000.00 ./${RSYSLOG_DYNNAME}.input
+
+startup
+
+# sleep a little to give rsyslog a chance to begin processing
```

Comment:
this sleep should no longer be necessary with the current testbench plumbing; I understand it is carried over from an older test. If you like, I can update one of the tests to the newest standards (I could also patch them after merge, so this is not a show stopper).